### PR TITLE
chore: add GitHub to titles

### PR DESCRIPTION
**Pull request by SkyeYoung**
Changes: 
add GitHub to titles

### CODE REVIEW

**Comment by SkyeYoung** [comment]
cc @guoqqqi

File: public/index.html [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <meta name="theme-color" content="#000000" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+  <!-- Google Tag Manager -->
```
Comment (juzhiyuan) [inline public/index.html:7]: Please don't format this part 😉

Comment (juzhiyuan) [reply public/index.html:7]: As they're copied from Google directly

File: public/index.html [hunk 2]
```diff
@@ -81,5 +74,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (guoqqqi) [inline public/index.html:79]:
```suggestion
</html>

```

**Comment by guoqqqi** [comment]
and need update the top title
image

File: public/index.html (hunk 1 above)
Comment (SkyeYoung) [reply public/index.html]: reverted

File: public/index.html (hunk 2 above)
Comment (SkyeYoung) [reply public/index.html]: reverted

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**